LaTeX/README.md

```diff
@@ -34,3 +37,6 @@ Now you should pick a LaTeX editor, here are some to choose from:
   5. [LyX](http://www.lyx.org/)
   6. [TeXlipse](http://texlipse.sourceforge.net/)
   7. [Sublime Text 3 - LaTeXTools](https://github.com/SublimeText/LaTeXTools)
+  8. [Visual Studio Code - LaTeX-Workshop](https://github.com/James-Yu/LaTeX-Workshop)
+
+> **Note**: You may find useful a PDF reader that watches your PDF files, like Skim `brew cask install skim` (Preferences -> Sync).
```

Comment:
This sentence may a small edit, I think a word is missing.

Reply:
@simeg Not sure how to word it better in order to make the intent clear.

> You may find useful a PDF reader that watches your PDF files, like Skim (can be installed by `brew cask install skim` and Sync feature can be enabled by (Preferences -> Sync)).

Reply:
Ah, I see now why I thought it was missing a word.

I find this sentence hard to read:
>You may find useful a PDF reader [..]

Could we re-word it to something like this?
>You may want to use a PDF reader [..]

Sorry for nitpicking.

Reply:
No big deal :), thank you.